book.rb

```diff
@@ -0,0 +1,17 @@
+require_relative 'rental'
+
+class Book
+  def initialize(title, author)
+    @title = title
+    @author = author
+    @rentals = []
+  end
+
+  attr_accessor :title, :book
+
+  attr_reader :rentals
```

Comment:
-  [ ] great job on setting up your associations 👏 in this line you're setting up the `attr_accessor` for `@title` and `@book` I think you might have made a typo here as the `attr_accessor` should be set for `@author` and `@title`, for `@book` it is not necessary.
- [ ] for `@rentals` you might want to setup both the setter and the getter as well.